README.md

```diff
@@ -36,6 +36,13 @@ repos:
     # Check if minimize id changes is selected in the plc project file.
     # See https://www.youtube.com/watch?v=KKpBtaYjfWo&t=935s why to do this.
     -   id: minimize-id-changes
+    # Checks if TwinCAT versions match in different tsproj files, or if it matches the targeted one.
+    -   id: check-twincat-versions
+        # Possible optional arguments
+        # --target-version: Set a version that you want the tsproj file to have
+        # --fix: Fix the version numbers if a target version is set
+        # --reason: Add a reason to the error message in case of a non-matching version.
+        args: [--target-version="3.1.4024.20", --fix, --reason="This version has a crucial new feature"]
```

Comment:
When I run with a version arg in quotes as shown here, the code fails to match versions properly and tries to fix it by placing an additional set of quotes, breaking the xml:
```
-<TcSmProject xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://www.beckhoff.com/schemas/2012/07/TcSmProject" TcSmVersion="1.0" TcVersion="3.1.4024.35" TcVersionFixed="true">
+<TcSmProject xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://www.beckhoff.com/schemas/2012/07/TcSmProject" TcSmVersion="1.0" TcVersion=""3.1.4024.35"" TcVersionFixed="true">
```

If try this without the quotes things work as expected and my files can be checked/fixed without issues.
```
-<TcSmProject xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://www.beckhoff.com/schemas/2012/07/TcSmProject" TcSmVersion="1.0" TcVersion="3.1.4024.35" TcVersionFixed="true">
+<TcSmProject xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://www.beckhoff.com/schemas/2012/07/TcSmProject" TcSmVersion="1.0" TcVersion="3.1.4024.50" TcVersionFixed="true">
```
```suggestion
        args: [--target-version=3.1.4024.20, --fix, --reason="This version has a crucial new feature"]
```